R/helpers.R

```diff
@@ -84,10 +84,11 @@ log_eval <- function(expr, level = TRACE, multiline = FALSE) {
 #' @examples
 #' log_separator()
 #' log_separator(ERROR, '!', width = 60)
```

Comment:
Can you please update the example using the separator without naming that argument and using positional matching -- that is off here due to having `namespace` as the 2nd arg now? and run `roxygenize`, or let me know if I should do these updates.

Thanks a lot for catching and fixing this  :+1: